HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -1203,7 +1205,7 @@ def create_schedule_file(col_name:,
     end
 
     if @schedules[col_name].nil?
-      @runner.registerError("Could not find the '#{col_name}' schedule.")
+      puts "Could not find the '#{col_name}' schedule."
```

Comment:
If we hit this, does it suggest that we have a bug in our code? If so, just convert to `fail` so that we see the bug and fix it.

Reply:
No. I changed this because you can now specify a `SchedulesFilePath` but you don't need to have every column in it.

Reply:
Perhaps we issue a warning?

Reply:
I assumed that, going forward, this method should only be called on columns that exist -- and if it accidentally got called on a column that didn't exist, that would represent a bug in our code. But you're saying that's not the case?

Reply:
So should we check for the column's existence outside of the `create_schedule_file` call? Or how else do we know whether we can call `create_schedule_file` for a given column?

Reply:
Yes, I would assume so. We'll need to know whether to call this method and create a schedule file, or call other methods dealing with weekday/weekend/multiplier, no?